ccm_web/server/src/api/api.service.ts

```diff
@@ -9,22 +9,30 @@ import { SectionApiHandler } from './api.section.handler'
 import { handleAPIError, makeResponse } from './api.utils'
 import { SectionUserDto } from './dtos/api.section.users.dto'
 import {
-  CanvasCourse, CanvasCourseBase, CanvasCourseSection, CanvasCourseSectionBase, CanvasEnrollment,
+  CanvasCourse,
+  CanvasCourseBase,
+  CanvasCourseSection,
+  CanvasCourseSectionBase,
+  CanvasEnrollment,
+  CanvasUser,
   CourseWithSections
 } from '../canvas/canvas.interfaces'
 import { CanvasService } from '../canvas/canvas.service'
 import { User } from '../user/user.model'
 
 import { Config } from '../config'
 import baseLogger from '../logger'
+import { InvitationService } from '../invitation/invitation.service'
+import { SectionExternalUserDto } from './dtos/api.section.external.users.dto'
```

Comment:
Nit-pick: we could group these with the other "NestJS module" imports above.

Reply:
Fixed.